### ISSUE TITLE
Narrator announces buttons inside ContentAligmentEditor incorrectly.

### ISSUE THREAD

**Issue by NikitaSemenovAkvelon** [dotnet/winforms#7800]
- .NET Core Version:
8.0.0-alpha.1.22467.6

**Probled description:**
Narrator announces all options as "selected".
![pPuKbzIH1f (1)](https://user-images.githubusercontent.com/109065597/191004377-6252ceda-fbfa-4522-8667-121cd51a66e3.gif)

**Expected behavior:**
Only for selected option it should announce "selected".

_Originally posted by @Tanya-Solyanik in https://github.com/dotnet/winforms/pull/7737#pullrequestreview-1109826285_

**Comment by Tanya-Solyanik** [comment]
It was @dmitrii-drobotov's finding 😄

**Comment by Tanya-Solyanik** [comment]
But this looks "by design" because we can also see the dotted rectangle around the button being announced.

**Comment by NikitaSemenovAkvelon** [comment]
If it is "by design" then we have a different behavior between `ContentAlignmentEditor` and `DockEditor` despite there being pretty similar.

**Comment by Tanya-Solyanik** [comment]
I see. In Office "number format" they have selection indicator and keyboard focus indicator - 
![Outlook number format drop down](https://user-images.githubusercontent.com/15823268/191171444-e98b13c9-6750-4015-bd14-7f0f89f0a8b7.png)

Here is narrator text in response to me opening the drop down and moving from the 1,2,3 format to `none`. `1,2,3` format is not announced as selected even though it would had been selected if I pressed enter on it.

> Numbering, Menu, 
>Recently Used Number Formats, 1, 2, 3, …, ALIGNMENT Left, Number Format: 1., <say-as interpret-as="spell">1</say-as> of 11, 
>Numbering Library, None, <say-as interpret-as="spell">3</say-as> of 11, **selected**, 
>1, 2, 3, …, ALIGNMENT Left, Number Format: 1., <say-as interpret-as="spell">4</say-as> of 11, 
>a, b, c, …, ALIGNMENT Left, Number Format: a), <say-as interpret-as="spell">8</say-as> of 11, 

Our UI does not had a gesture that represents selection(enter key) and it does not have visual indicator either.

**Comment by dmitrii-drobotov** [comment]
>but we don't have a visual representation of the current selection. We could color in the selected rectangle.

Unlike Office in WinForms there is a text on a ComboBox with current value, so in a way we already have a representation. But I agree that adding another indication could be useful. 

For this issue though I think we can only fix the problem with "selected" state announcements for every option, so that it would announce "selected" only for actually selected value and "non-selected" for other.

**Comment by Tanya-Solyanik** [comment]
> Unlike Office in WinForms there is a text on a ComboBox with current value, so in a way we already have a representation. But I agree that adding another indication could be useful.

Agree.

>so that it would announce "selected" only for actually selected value and "non-selected" for other.

Agree.

**Comment by Philip-Wang01** [comment]
Verified this issue with latest .NET 8.0.100-alpha.1.22512.27, issue is fixed. Gif as below:

![testrfix](https://user-images.githubusercontent.com/86937911/195524148-6bf96bda-01f5-486b-a1b5-5c76f93ca749.gif)
